packages/shared/src/lib/featureValues.ts

```diff
@@ -9,7 +9,7 @@ export enum ExperimentWinner {
 export enum PlusPriceType {
   Monthly = 'monthly',
   Yearly = 'yearly',
-  OneTimePayment = 'one-time',
+  GiftOneYear = 'gift_one_year',
```

Comment:
@rebelchris is this ok? If I remember correctly strings that are not "monthly" or "yearly" causes errors 👀

Reply:
@sshanzel yeah Luca is right I think this will break the API if you test it.
Did you try and test it locally and get the webhook response?

Reply:
Do you guys suggest anything? Haven't tried the actual Paddle checkout with this one.

Reply:
You'd have to try it out or do API first to handle these gifted users?
But mabye just set it to Yearly as well as that's technically it?

Reply:
Yeah, we can just use yearly.

Reply:
gift_one_year seem to have worked. Also received an email. Is there any part I should check? Having it separated makes it easier to manage.

![Screenshot 2025-01-28 at 8 02 38 PM](https://github.com/user-attachments/assets/99abf6af-36b6-49ae-a89c-06a28f2677c6)